app/controllers/admin/pods_controller.rb

```diff
@@ -0,0 +1,21 @@
+
+module Admin
+  class PodsController < AdminController
+    respond_to :html, :json
+
+    def index
```

Comment:
I'm really not convinced this ABC size check improves code quality. Maybe we could raise the value to something a bit saner?